content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,25 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Fixed
+
+- We finished migrating all tenants from East/West into GovCloud! Thank you to all our customers for their patience and cooperation in making this move successful.
```

Comment:
Hmm, good question. I feel like I remember @mogul celebrating everybody being out of E/W this time around. Is it that we _verified_ all customers are out of E/W and can begin shutting it down?